content/en/docs/instrumentation/js/getting-started/nodejs.md

```diff
@@ -19,11 +21,16 @@ Ensure that you have the following installed locally:
 
 ## Example Application
 
-The following example uses a basic Express application.
+The following example uses a basic [Express](https://expressjs.com/)
+application. If you're not using Express, that's fine — this guide will also
+work with Koa, Nest.JS
+[and more](/ecosystem/registry/?component=instrumentation&language=js)
+
+For more elaborate examples, see [examples](/docs/instrumentation/js/examples/).
 
 ### Dependencies
 
-First, create an empty package.json:
+To begin, set up an empty package.json in a new directory:
 
 ```shell
 npm init -f
```

Comment:
-f is typically not recommended

Reply:
I don't want people to go through all the prompts `npm init` gives you just for going through the tutorial. Reading through the help of `npm init` real quick an at least safer version of it is `npm init -y`:

yes: `Automatically answer "yes" to any prompts that npm might print on the command line.` 

force: `Removes various protections against unfortunate side effects, common mistakes, unnecessary performance degradation, and malicious input.`

Would that work?

Reply:
yeah that seems reasonable

Reply:
```suggestion
npm init -y
```